### PR TITLE
fix delay queue saving

### PR DESCRIPTION
**Pull request by medliii**

### CODE REVIEW

File: src/Service/QueueService.php [hunk 1]
```diff
@@ -125,7 +125,8 @@ public function getToRepublish(int $limit, ?int $offset = null): array
      */
     public function createQueue(QueueInterface $queueable): QueueEntityInterface
     {
-        return $this->entityFactory->createQueue($queueable);
+        $queue = $this->entityFactory->createQueue($queueable);
+        return $this->save($queue);
```
Comment (omnilight) [inline src/Service/QueueService.php]: Надо дополнить тест https://github.com/lamoda/queue-bundle/blob/master/tests/unit/Service/QueueServiceTest.php

Comment (medliii) [reply src/Service/QueueService.php]: > Надо дополнить тест https://github.com/lamoda/queue-bundle/blob/master/tests/unit/Service/QueueServiceTest.php

Написал

File: tests/unit/Service/QueueServiceTest.php [hunk 2]
```diff
@@ -183,6 +188,39 @@ public function testCreateQueue(): void
...
+        $queueRepository = $this->getQueueRepository();
+        $queueRepository
+            ->expects($this->once())
+            ->method('save')
+            ->with($queue)
+            ->willReturnArgument(0);
+
+        $entityFactory = $this->getEntityFactory();
+        $entityFactory
+            ->expects($this->once()) #at least once
```
Comment (Tekill) [inline tests/unit/Service/QueueServiceTest.php]: `$this->once` is only once, not at least

File: src/Service/QueueService.php (hunk 1 above)
Comment (Tekill) [reply src/Service/QueueService.php]: Add a blank line before return statements, unless the return is alone inside a statement-group (like an if statement);

https://symfony.com/doc/current/contributing/code/standards.html

**Comment by akurov-lam** [comment]
Коммиты надо схлопнуть, наверное

**Pull request merged**